src/jobs/test.yml

```diff
@@ -0,0 +1,33 @@
+description: |
+  Simple drop-in job to test your PHP application automatically.
+
+parameters:
+  version:
+    type: string
+    description: The `cimg/php` Docker image version tag.
```

Comment:
No default here means this is mandatory, which doesn't quite align with the executor. Should have a default.

Reply:
done in https://github.com/TangRufus/php-orb/commit/d13f956682310bf798121de056b7eb124e624677